js/main.js

```diff
@@ -39,3 +39,124 @@ const getRandomDecimal = (min, max, precision) => {
 
 getRandomDecimal(0, 5, 2);
 getRandomInt(0.5, 2.5);
+
+
+const AUTHORS = ['Aras Mayer', 'Ehsan Bain', 'Fearne Armstrong',
+  'Yasmine Contreras', 'Amisha Figueroa', 'Harper-Rose Sparrow', 'Kyran Boyce',
+  'Tia Roth', 'Ayse Haynes', 'Yahya Mckee', 'Ananya Manning', 'Larry Wheatley',
+  'Alaw Washington', 'Amelia Mackenzie', 'Yazmin Smart', 'Awais Hawes'];
+
+const OFFER_TITLE = ['Oakwood Premier Tokyo', 'Hotel ICHIGAYA（JR Ichigaya Sta.)',
+  'TOE LIBRARY Tokyo Asakusa', 'SCASA NishiShinjuku', 'COTO Tokyo Shibuya 4',
+  'Studio Inn Nishi Shinjuku', 'COTO Tokyo Shibuya 2',
+  'Stay SAKURA Tokyo Shinjuku Hyaku Kura', 'Hotel Axas Nihonbashi',
+  'Space F 代々木', 'Hermitage Nishi Shinjuku'];
```

Comment:
```suggestion
const OFFER_TITLE = [
  'Oakwood Premier Tokyo', 'Hotel ICHIGAYA（JR Ichigaya Sta.)',
  'TOE LIBRARY Tokyo Asakusa', 'SCASA NishiShinjuku', 'COTO Tokyo Shibuya 4',
  'Studio Inn Nishi Shinjuku', 'COTO Tokyo Shibuya 2',
  'Stay SAKURA Tokyo Shinjuku Hyaku Kura', 'Hotel Axas Nihonbashi',
  'Space F 代々木', 'Hermitage Nishi Shinjuku'
];
```

Если у тебя много много данных, на много много строк, то лучше вот ак писать. причем, ниже ты пишешь в таком стиле, а тут ты пишешь не в другом. Надо писать однообразно.

Reply:
ESLint ругался что нет отступа перед строками, и пришлось их собрать обратно